doc/user/bmp.rst

```diff
@@ -57,6 +57,51 @@ The `BMP` implementation in FRR has the following properties:
 
 - monitoring peers with :rfc:`5549` extended next-hops has not been tested.
 
+RFC 7854
+========
+Unsupported features (non exhaustive):
+  - Per-Peer Header
+    
+    - Peer Type Flag
+    - Peer Distingsher
+  
+  - Peer Up
+  
+    - Reason codes (according to TODO comments in code)
+
+Peer Type Flag and Peer Distinguisher can be implemented easily using RFC 9069's base code.
+    
+RFC 9069
+========
+Everything that isn't listed here is implemented and should be working.
+Unsupported features (should be exhaustive):
+
+- Per-Peer Header
+
+  - Timestamp
+
+    - set to 0
```

Comment:
as per previous commit https://github.com/FRRouting/frr/pull/11800/commits/4ea750578f944ebf1e7a195589f67c4512e819c2 handles timestamps, I think this assertion is not correct now.

Reply:
no, the rib timestamp is still WIP and as stated in the commit message the `value not set in the msg yet, testing and double checking is needed before` 🙂 
you can double check that it's true in the code if u want :
https://github.com/FRRouting/frr/pull/11800/files#diff-af2d89aaf8862c57f4f02385cb1c5850844025ed3710d9be119a8b98af85d0ceR365